### DISCUSSION TITLE
Regarding spot instance

### DISCUSSION THREAD

**Discussion by devopsmukund** [learndevopsonline/learndevopsonline#413] [Q&A] [answered]
Hi Raghu
I couldn't find the spot instance file in the documents you provided, where can i find that or you can send me that..

**Comment by noolivara** [comment, marked as answer]
@devopsmukund, 
https://learndevopsonline.github.io/learndevopsonline/build/docs/How-Tos/increase-spot-quota

https://learndevopsonline.github.io/learndevopsonline/build/docs/How-Tos/show-name-tag-in-spot-request
